sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-soname.patch

```diff
@@ -5,7 +5,7 @@
  fragile_SHARED_SRC_LIST=@FRAGILE_SHARED_SRC_LIST@
  libfastjetcontribfragile.@DYNLIBEXT@: $(fragile_SHARED_SRC_LIST)
 -	$(CXX) @DYNLIBOPT@ -fPIC -DPIC $(CXXFLAGS) `$(FASTJETCONFIG) --cxxflags --libs` $(fragile_SHARED_SRC_LIST) -o libfastjetcontribfragile.@DYNLIBEXT@
-+	$(CXX) @DYNLIBOPT@ -fPIC -DPIC -Wl,-Ol -Wl,--as-needed -Wl,-soname,fastjetcontribfragile.so.0 $(LDFLAGS) $(CXXFLAGS) `$(FASTJETCONFIG) --cxxflags --libs` $(fragile_SHARED_SRC_LIST) -o libfastjetcontribfragile.@DYNLIBEXT@
++	$(CXX) @DYNLIBOPT@ -fPIC -DPIC -Wl,--as-needed -Wl,-soname,fastjetcontribfragile.so.0 $(LDFLAGS) $(CXXFLAGS) `$(FASTJETCONFIG) --cxxflags --libs` $(fragile_SHARED_SRC_LIST) -o libfastjetcontribfragile.@DYNLIBEXT@
```

Comment:
What is the reason for this? What does `-Ol` do?

Reply:
I think that was a typo. `-Ol` was probably a typo of `-O1`.

Reply:
I could not find anything on `-Ol`, surprising that this did not give an error...

Reply:
Maybe the linker just ignores `-O[non-numeric]` for future compatibility.

Reply:
> I could not find anything on `-Ol`, surprising that this did not give an error...

Yeah me neither, which is why I was super confused.